charts/pelorus/charts/exporters/templates/_deploymentconfig.yaml

```diff
@@ -32,6 +32,12 @@ spec:
           configMap:
             name: {{ $config.map_name }}
         {{- end }}
+        {{- if .custom_ca_configmap }}
+        - name: configmap-cluster-ca-bundle
+          configMap:
+            name: {{ .custom_ca_configmap }}
+            defaultMode: 420
```

Comment:
Based on the doc, isn't this part missing `items` ?

https://docs.openshift.com/container-platform/4.16/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki

Reply:
This is just the pod `volumes:` definition pointing to a ConfigMap, no need to iterate here since we need just one. The ConfigMap can be created following the steps in the docs you linked.

Reply:
I refer to that part of the doc:

```
      - name: trusted-ca
        configMap:
          name: trusted-ca
          items:
            - key: ca-bundle.crt 
              path: tls-ca-bundle.pem 



ca-bundle.crt is required as the ConfigMap key.
tls-ca-bundle.pem is required as the ConfigMap path.

```

Reply:
Ah now I got it. I refactored that part to be consistent with the documentation. It was working anyway because I chose another mount point where it was not required to rename the key.